src/log.c

```diff
@@ -271,7 +280,7 @@ static const char* level_name(int level) {
 static const char *SEPARATORS=", ;:";
 
 /// Calculate the bit mask for a log enable string. Return -1 and set qd_error on error.
-static int enable_mask(const char *enable_) {
+static int enable_mask(const char *enable_, bool log_error) {
```

Comment:
This also appears to only be called with log_error=false, which makes the log_error parameter & logic not necessary.